include/odp/api/spec/dma_types.h

```diff
@@ -492,6 +535,77 @@ typedef struct odp_dma_transfer_param_t {
 	 */
 	odp_dma_seg_t *dst_seg;
 
+	/** Destination segment pool
+	 *
+	 *  When odp_dma_transfer_param_t::dst_alloc is set, provide the pool details from which
+	 *  the buffers need to be allocated.
+	 */
+	odp_pool_t dst_seg_pool;
+
+	/** Transfer hints
+	 *
+	 *  Depending on the implementation, adjusting hints bit fields may improve performance.
+	 *  Initialize all unused bits to zero.
+	 */
+	union {
+		/** Hint bit fields */
+		struct {
```

Comment:
These aren't actually hints anymore, they are more like options that applications should properly define per-transfer for optimal performance, so perhaps a few adjustments:
- "Transfers hints" -> "Transfer options"
- The sentence "Depending on the implementation, adjusting hints bit fields may improve performance." could be changed to "Options to further configure the transfer." or something similar
- "Hint bit fields" -> "Option bit fields"
- Change "hint/hints" to "option/options" in the definitions below, e.g. "Hint bit to denote a source segment ..." -> "Option bit to denote a source segment ..." etc.
- The `struct` could be named to "opts", so something like this:
```C
...
union {
    ...
    struct {
        ...
    } opts;
    ...
    uint16_t all_opts;
};
```